.github/policies/resourceManagement.yml

```diff
@@ -54,7 +54,7 @@ configuration:
           days: 7
       actions:
       - addReply:
-          reply: Hi @${issueAuthor}, since you haven’t asked that we “`/unresolve`” the issue, we’ll close this out. If you believe further discussion is needed, please add a comment “`/unresolve`” to reopen the issue.
+          reply: Hi @${issueAuthor}, since you haven't asked that we "`/unresolve`" the issue, we'll close this out. If you believe further discussion is needed, please add a comment "`/unresolve`" to reopen the issue.

```

Comment:
Is this tested? If not we should test it after merging the PR - I wonder if the single and double quotes need to be escaped

Reply:
The encoding looks like not comply with English chars. So I updated all of them. Will test after merging to main.